source/features/link-to-compare-diff.tsx

```diff
@@ -0,0 +1,27 @@
+import './link-to-compare-diff.css';
+import React from 'dom-chef';
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import {wrapAll} from '../helpers/dom-utils';
+
+function init(): void {
+	const changedFilesSummary = select('.Box .octicon-file-diff')!.closest('li')!;
+	wrapAll(
+		[...changedFilesSummary.children],
+		<a className="Link--muted" href="#diff"/>,
```

Comment:
Looks like it was needed for the blue hover actually.

Reply:
Nope, that blue hover is added by our own CSS

I guess you didn't get that color because you didn't do 341b778d8839162a48d9a5c1671ee478fe7045f8 _first_.